mail/notmuch/Portfile

```diff
@@ -5,6 +5,7 @@ PortGroup           conflicts_build 1.0
 
 name                notmuch
 version             0.29
+revision            2
```

Comment:
Why 2 ? If the previous revision was 0 then just bump it to 1.

Reply:
Ah, I fiddled with two revisions locally and forgot to turn it back to 1 before opening the PR. I'll change it.